### ISSUE TITLE
Request: Improve output of plan-all with landscape

### ISSUE THREAD

**Issue by flmmartins** [gruntwork-io/terragrunt#726]
Hello All,

Would be a possible to integrate plan-all with https://github.com/coinbase/terraform-landscape ?
That would make output better and if there was a way to integrate it would be nice. 
Usually I use it with:

terragrunt plan | landscape

But this only works without the -all.

Best,

**Comment by yorinasub17** [comment]
That sounds useful! If anyone is up for a PR that improves the `plan-all` output to support this, would be much appreciated.

**Comment by flmmartins** [comment]
Do you accept open source contributions? Is there any guidelines?

**Comment by yorinasub17** [comment]
Yup we do! Take a look at [Developing Terragrunt](https://github.com/gruntwork-io/terragrunt#developing-terragrunt) in the README for some of our guidelines. In general, it boils down to:

- If possible, share design in issues before tackling the work. It helps to make sure the implementation details align with the contributors.
- Run code formatters.
- Make sure to add tests.
- Make sure tests pass.

**Comment by brikis98** [comment]
See also https://github.com/gruntwork-io/terragrunt/pull/762

**Comment by ryno75** [comment]
in light of Terraform v0.12's enhanced output formatting and it's incompatibility with terraform-landscape... this seems like a moot issue at this point.
